src/pages/Subreddit/Subreddit.js

```diff
@@ -0,0 +1,48 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import './Subreddit.css'
+
+
+function Subreddit() {
+    const { id } = useParams();
+    const [childReddit, setChildReddit] = useState({})
+
+    useEffect(()=>{
+    async function fetchSub() {
+        try {
+            const response = await axios.get(`https://www.reddit.com/r/${id}/about.json`);
+            setChildReddit(response.data.data)
+
+        } catch (e) {
+            console.error(e);
+        }
+    }
+
+
+    fetchSub();
+    },[])
+    console.log(childReddit)
+    return (        <div>
```

Comment:
Je zult dan hiervoor wel nog een check moeten toevoegen
```suggestion
    if (!childReddit) return null;

    return (        <div>
```